doc_source/appconfig-creating-deployment-strategy.md

```diff
@@ -171,4 +170,4 @@ The following procedure describes how to use the AWS CLI \(on Linux or Windows\)
    ResponseMetadata            : Runtime Metadata
    ```
 
-------
```

Comment:
Sorry about this change. I did not do it myself. I guess the github code editor added/removed the new line character at end of file automatically.
Let me know if you want me to revert this change by downloading the code and doing the change in some proper IDE. Or if you want to just do the change directly in your already setup environment from scratch I don't mind that either. It's a really minor change anyway.